src/main-client.ts

```diff
@@ -337,27 +631,87 @@ export class MainClient extends BaseRestClient {
     return this.getPrivate('sapi/v1/account/apiRestrictions');
   }
 
-  acceptQuoteRequest(params: AcceptQuoteRequestParams): Promise<any> {
-    return this.postPrivate('sapi/v1/convert/acceptQuote', params);
-  }
-
   enableConvertSubAccount(params: EnableConvertSubAccountParams): Promise<any> {
     return this.postPrivate('sapi/v1/broker/subAccount/convert', params);
   }
 
-  convertQuoteRequest(params: ConvertQuoteRequestParams): Promise<any> {
-    return this.postPrivate('sapi/v1/convert/getQuote', params);
```

Comment:
What happened to these? They don't work anymore?

Reply:
What do you mean?

Reply:
They are still in the docs.

Reply:
This commit seems to be removing this function, or did you rename the method? We shouldn't rename existing methods because that's a breaking change for anyone that's already using it.

Reply:
Fixed! Reverted back, must be an accident while I tried to move these two functions. 

I will move all the functions around when we finish this PR.

Reply:
> I will move all the functions around when we finish this PR.

in a separate PR please 😁